test/expect-tests/persistent_tests.ml

```diff
@@ -0,0 +1,41 @@
+open Stdune
+module Persistent = Dune_util.Persistent
+module Digest = Dune_digest
+
+let test (type a) (module Persistent : Persistent.Desc with type t = a)
+    (example : a) =
+  let digest = Digest.generic example |> Digest.to_string in
+  printfn "%s version %d\n%s\n---\n" Persistent.name Persistent.version digest
+
+let%expect_test "persistent digests" =
```

Comment:
Could you add some comments to this file instructing what to do (bump the version) when a test fails?